README.md

```diff
@@ -1,40 +1,43 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Hello Visitor !! Welcome in PodCodar web App
 
-[![image](./public/images/vercel-logo.svg)][1]
+It's nice have you for here! This is PodCodar web a project of the community PodCodar with it's main objective is to teach good practices of development for our mentees.
 
-## Getting Started
+### Technologies
 
-First, run the development server:
+<p align="left">
+<a href="https://www.typescriptlang.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/typescript/typescript-original.svg" alt="typescript" width="40" height="40" paddingLeft="5px"/>  </a>
```

Comment:
Usually, keeping links like this can be boring, because, if the original source stop serving this file it will appear broke in our documentation, demanding new sources and changes in the doc.

As a suggestion, I believe you can download the images you're using, and we can create a folder like `docs/images` where we can upload those images and free ourselves from this condition^^

Reply:
Great suggestions.

Another thing to avoid is HTML tags. We should always preferer Markdown notation over HTML tags